packages/application-shell/src/apollo-links/token-retry-link.ts

```diff
@@ -18,34 +19,46 @@ type ApolloContext = {
 // so that the MC BE can issue a new token.
 // NOTE: the retry is not meant to work for the MC access token.
 
+const TOKEN_RETRY_HEADER_NAME = 'X-Force-Token';
+
 export const getDoesGraphQLTargetSupportTokenRetry = (
-  requestHeaders: ApolloContext['headers']
-) => {
-  const target = (requestHeaders['X-Graphql-Target'] ||
-    requestHeaders['x-graphql-target']) as TokenRetryGraphQlTarget;
+  context: ApolloContext
+): boolean => {
+  const target = (context.headers['X-Graphql-Target'] ||
+    context.headers['x-graphql-target']) as TokenRetryGraphQlTarget;
```

Comment:
Also here, we can use the constant

```suggestion
  const target = (context.headers[SUPPORTED_HEADERS.X_GRAPHQL_TARGET] ||
    context.headers[SUPPORTED_HEADERS.X_GRAPHQL_TARGET.toLowerCase()]) as TokenRetryGraphQlTarget;
```